compiler/src/dmd/funcsem.d

```diff
@@ -154,6 +154,28 @@ public:
     }
 }
 
+/****************************************
+ * Only one entry point function is allowed. Print error if more than one.
+ * Params:
+ *      fd = a "main" function
+ * Returns:
+ *      true if haven't seen "main" before
+ */
+extern (C++) bool onlyOneMain(FuncDeclaration fd)
```

Comment:
not sure where in the headers this should go, or for that matter where in DMD. This code is duplicated by LDC and GDC, hence the `extern(C++)`.